### ISSUE TITLE
Notebooks home page

### ISSUE THREAD

**Issue by benvenker** [sourcegraph/sourcegraph#28587]
- Search notebooks (public and private)
- Sections for your notebooks and favorited notebooks
- Sort by favorites
- Sort by views

Figma sketch is [here](https://www.figma.com/file/32wyz09t6u2bSBvsJwmDEA/Search-Notebooks-GA?node-id=1324%3A4932) and should be enough to unblock basic frontend development (but let @benvenker know if it's not).

**Comment by benvenker** [comment]
Already done
